styleguide/source/_patterns/02-organisms/05-ec/01-ec-search-bar/ec-search-bar.twig

```diff
@@ -1,10 +1,8 @@
-<div class="ec-search-bar">
-  <div class="ec-search-bar-inner">
-    <div class="ec-search-bar_library">
-      {% include "atoms-button-link-secondary" with {'content': 'Library'} %}
-    </div>
-    <div class="ec-search-bar_form">
-      {% include "molecules-search-field" with {'placeholder': 'Search Courses'} %}
-    </div>
+<div class="ec_search-bar_inner layout">
```

Comment:
I'm not sure I understand the refactoring here. It's one thing to refactor the pattern to use `<sections>` rather than `<div>` but quite another to fragment the pattern, which is what has happened here. Please return the parent container with the class `ec_search-bar` to the pattern.